packages/core/src/html/ComponentProcessor.js

```diff
@@ -0,0 +1,162 @@
+const cheerio = require('cheerio');
+
+const { findHeaderElement } = require('./headerProcessor');
+const { getVslotShorthandName } = require('./vueSlotSyntaxProcessor');
+const { warnConflictingAttributes, warnDeprecatedAttributes } = require('./warnings');
+
+const _ = {};
+_.has = require('lodash/has');
+
+const logger = require('../utils/logger');
+
+class ComponentProcessor {
+  constructor(markdownProcessor) {
+    this.markdownProcessor = markdownProcessor;
+  }
+
+  /*
+   * Panels
+   */
+
+  processPanelAttributes(node) {
+    this.markdownProcessor.processAttributeWithoutOverride(node, 'alt', false, '_alt');
+    this.markdownProcessor.processAttributeWithoutOverride(node, 'header', false);
+  }
+
+  /**
+   * Assigns an id to the root element of a panel component using the heading specified in the
+   * panel's header slot or attribute (if any), with the header slot having priority if present.
+   * This is to ensure anchors still work when panels are in their minimized form.
+   * @param node The root panel element
+   */
+  static assignPanelId(node) {
```

Comment:
how about `headerProcessor.js`? seems closely related

Reply:
Yup, makes sense to be in `headerProcessor.js`